analysis/aws_rules_vpc_flow_logs/aws_vpc_unapproved_outbound_dns.yml

```diff
@@ -1,6 +1,6 @@
 AnalysisType: rule
 Filename: aws_vpc_unapproved_outbound_dns.py
-PolicyID: AWS.VPCFlowLog.UnapprovedOutboundDNS
```

Comment:
Maybe `AWS.VPCFlow.X` for these? I like distinguishing that these are flow logs but maybe that's unnecessary. I could go either way on this.